library/src/main/java/com/pokegoapi/main/CommonRequest.java

```diff
@@ -104,4 +108,50 @@ public static GetInventoryMessage getDefaultGetInventoryMessage(PokemonGo api) {
 				CommonRequest.getDownloadSettingsMessageRequest(api));
 		return serverRequests;
 	}
+
+
+	/**
+	 * List of common requests
+	 *
+	 * @param api The current instance of PokemonGO
+	 * @return an array of ServerRequest
+	 */
+	public static ServerRequest[] commonRequests(PokemonGo api) {
+		ServerRequest[] serverRequests = new ServerRequest[4];
```

Comment:
yes... but if we use the existent ebum we cant use them to return an array, ordinal value arent what will be used..on request handler its important to maintain the order of requests. we can convert into an enum map to make it even less readable if you want :)